Source/SwiftIcons.swift

```diff
@@ -672,8 +676,10 @@ public extension UIBarButtonItem {
         
         title = nil
         let button = UIButton(frame: cgRect)
+        let onTapColor = color.withAlphaComponent(0.4)
```

Comment:
`color` is not used in this extension. Please, change it to `iconColor`.

This failed travis build.